### PR TITLE
Add some comments for Server re-org sample

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

**Review by osfameron — COMMENTED** [review]
Same comment as https://github.com/couchbase/docs-sdk-nodejs/pull/155

I'm not sure of the value of repeating the comment here, or the level of abstraction of these particular comments? (Would be great to go over in our session next week!)

File: modules/hello-world/examples/n1ql_hello_world.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 
 try:
     # tag::n1ql-query[]
+    # To execute the query, we call the query() function on our cluster object and store the result.
```
Comment (simon-dew) [inline modules/hello-world/examples/n1ql_hello_world.py]:
```suggestion
    # Call the query() function on the cluster object and store the result.
```

File: modules/hello-world/examples/n1ql_hello_world.py [hunk 2]
```diff
@@ -6,7 +6,11 @@
...
     result = cluster.query("SELECT \"Hello World\" as greeting")
+
+    # The result object will include an array of rows found.
+    # We iterate over the rows to access our result data and print it to the terminal.
```
Comment (simon-dew) [inline modules/hello-world/examples/n1ql_hello_world.py]:
```suggestion
    # Iterate over the rows to access result data and print to the terminal.
```

**Review by simon-dew — COMMENTED** [review]
Same comments as for [couchbase/docs-sdk-nodejs#155](https://github.com/couchbase/docs-sdk-nodejs/pull/155)

**Pull request merged**